### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Create a Scala Project_Resize: On custom scaling to 200%, the contents of the screen are not visible

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4298]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.4.4; #Resize; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev1;

**NOTE:** Same issue is observed in below screen
Create a Scala Project > Next > Libraries tab

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**MAS Violated:** MAS 1.4.4

**Repro Steps:**
1.From the Display Settings > Scale and Layout > Advanced Scaling Settings > Custom Scaling, adjust the custom scaling to 200%
2.Open Java EE - Eclipse application 
3.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
4.Tab till "File > New > Project" in the navigation bar and press enter
5.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
6.Tab till "Next" button and press enter
7.Verify if all the contents of "Create a Scala Project" pop-up are visble

**Actual Result:**
On custom scaling to 200%, the contents of the "Create a Scala Project" pop-up are not visble

**Expected Result:**
On custom scaling to 200%, all the contents of the "Create a Scala Project" pop-up should be visble

**User impact:**
Users with visual impairments will not be able to ceate new Scala projects

**MAS Reference:**
[MAS 1.4.4](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B5BE8E205-7508-414C-A273-657770321DA0%7D&file=MAS%201.4.4%20%E2%80%93%20Resize%20Text.docx&action=default&mobileredirect=true)

[Bug_4298_Java Tooling_Create a Scala Project_Zooming.pptx](https://github.com/microsoft/azure-tools-for-java/files/4468519/Bug_4298_Java.Tooling_Create.a.Scala.Project_Zooming.pptx)

![Bug_4298_Java Tooling_Libraries_Zooming](https://user-images.githubusercontent.com/41108985/79102499-7d557c00-7d88-11ea-8db1-810cb5c970c8.png)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Eclipse wizard. You can find the same dialog in New Project > Java > Java Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse without our toolkit.
![image](https://user-images.githubusercontent.com/36756024/81458286-3a8e9480-91cc-11ea-9cf8-816140e2b174.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Important. Users with visual impairments will not be able to ceate new Scala projects
+ Quality of experience: Manageable. User could use narrator to read the button to perform further creat process.
+ Alternate solutions: User could use narrator to read the button to perform further create process.
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562983

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749649
